pkg/controller/importconfig/cluster_info.go

```diff
@@ -259,9 +261,26 @@ func getBootstrapKubeConfigData(ctx context.Context, clientHolder *helpers.Clien
 	}
 
 	kubeConifgData := getBootstrapKubeConfigDataFromImportSecret(importSecret)
+	kubeAPIServer, caData, token, err := parseKubeConfigData(kubeConifgData)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// check if the kube apiserver address is changed
+	validKubeAPIServer, err := validateKubeAPIServerAddress(ctx, kubeAPIServer, clientHolder)
```

Comment:
all these checking also need to get infrastructure and apiserver, why do not we just `return createKubeConfig(ctx, clientHolder, cluster)` directly?

Reply:
`createKubeConfig(ctx, clientHolder, cluster)` will create a new token. If the current bootstrap kubeconfig is valid, it is not necessary to create a new one, which will cause the restarting of the klusterlet agents.